README.md

```diff
@@ -58,24 +58,43 @@ final class AppDelegate: ApplicationDelegate {
 		```
   </details>
 - <details>
-    <summary>Vulkan SDK</summary>
+    <summary>Vulkan SDK [<--click here for instructions]</summary>
+
+	- Vulkan SDK from LunarG repository for Debian/Ubuntu based distributions, HIGHLY RECOMMENDED METHOD.
 
-	- Vulkan SDK from LunarG repository
 		LunarG is using deprecated apt-key to verify signature so this repo provides more modern and safe configuration via `SupportingFiles`
 		```bash
 		wget -qO - https://packages.lunarg.com/lunarg-signing-key-pub.asc | gpg --dearmor | sudo tee -a /usr/share/keyrings/lunarg-archive-keyring.gpg
 		sudo wget -q https://raw.githubusercontent.com/smumriak/AppKid/main/SupportingFiles/lunarg-vulkan-jammy.list -O /etc/apt/sources.list.d/lunarg-vulkan-jammy.list
 		sudo apt update
-		sudo apt install vulkan-sdk -y
-		```
-	- Alternativelly you can try to install Vulkan SDK from tarballs provided by LunarG or if you are using Debian 12 and newer or Ubuntu 22.04 and newer you can substitute Vulkan SDK from LunarG by installing 	libvulkan-dev
+		sudo apt install -y vulkan-sdk
 		```
-		sudo apt install libvulkan-dev
-		```
-		But you will have to manually install `glslc` compiler from Gooogle's shaderc project located [here](https://github.com/google/shaderc).
+	- Other places to get nearly-equivalent stuff [ for other system types, or if u can't use the above repo]:
+
+		- libvulkan
+			If you are using Debian 12+ or Ubuntu 22.04+ you can get most of what u need with 'libvulkan-dev' as a substitute for Vulkan SDK
```

Comment:
This is not quite true. vulkan-sdk is a meta package that depends on like 23 packages.

You can start this section with something like 
> As an alternative you can install parts of Vulkan SDK provided by your distribution or by installing them from some other source
and then list libvulkan-dev (it's important! libvulkan1 contains library only, libulkan-dev contains Vulkan headers) and glslc